Formula/g/giflib.rb

```diff
@@ -24,12 +25,8 @@ class Giflib < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d753208ed3a4bbd60b59e3ca4466196e4b935d4f434935b540fc6bfb5f3e0385"
   end
 
-  # Upstream has stripped out the previous autotools-based build system and their
-  # Makefile doesn't work on macOS. See https://sourceforge.net/p/giflib/bugs/133/
-  patch :p0 do
-    url "https://sourceforge.net/p/giflib/bugs/_discuss/thread/4e811ad29b/c323/attachment/Makefile.patch"
-    sha256 "a94e7bdd8840a31cecacc301684dfdbf7b98773ad824aeaab611fabfdc513036"
-  end
+  # remove gif file resizing step so that imagemagick is not a required dependency
```

Comment:
Can you submit this upstream?

Reply:
I submitted an [upstream merge request](https://sourceforge.net/p/giflib/code/merge-requests/16/) and filed a [bug](https://sourceforge.net/p/giflib/bugs/171/) to track this.

Reply:
I just noticed that @eric-s-raymond landed [Avoid having an install-time dependency on convert(1)](https://sourceforge.net/u/bvillaluz/giflib/ci/d54b45b0240d455bbaedee4be5203d2703e59967/) on 2/24, so my merge request isn't necessary. I updated the patch comment with a reference to @eric-s-raymond's fix.